components/constellation/constellation.rs

```diff
@@ -1867,6 +1870,15 @@ where
             ScriptToConstellationMessage::FinishJavaScriptEvaluation(evaluation_id, result) => {
                 self.handle_finish_javascript_evaluation(evaluation_id, result)
             },
+            ScriptToConstellationMessage::WebDriverInputComplete(msg_id) => {
+                if let Some(ref reply_chan) = self.webdriver.input_command_response_channel {
+                    reply_chan
```

Comment:
See [#29679](https://github.com/servo/servo/issues/29679):

```suggestion
                if let Some(ref reply_sender) = self.webdriver.input_command_response_channel {
                    reply_sender
```